src/web/handlers/NativeViewGestureHandler.ts

```diff
@@ -83,7 +83,11 @@ export default class NativeViewGestureHandler extends GestureHandler {
     }
 
     this.begin();
-    if (this.buttonRole) {
+
+    const view = this.delegate.getView() as HTMLElement;
+    const isRNGHText = view.hasAttribute('rnghtext');
```

Comment:
This one is a different case letters than they are set in the component.

Reply:
Yes, the thing is when those attributes are passed into HTML components, they are automatically lowercased. I'll change that in `Text` component such that both will be lowercased.

Reply:
Done in [31aba7a](https://github.com/software-mansion/react-native-gesture-handler/pull/3202/commits/31aba7a6e18b60ef23a190e5cb6dec3255229114)